test/TestDeprecatedModule.py

```diff
@@ -1,10 +1,9 @@
 import unittest
+from test import ANSIBLE_MAJOR_VERSION, RunFromText
 
 import pytest
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.